lib/experimental/new_embed.dart

```diff
@@ -161,13 +161,20 @@ abstract class TabView {
 }
 
 class EditorTabView extends TabView {
+  Timer _deduplicationTimer;
+
+  // TODO(RedBrogdon): Add UI Elements for errors and warnings indicators
```

Comment:
I was wondering what UI had been done in this PR. Good call just doing the editor part. I can pick this up and run with it.

Reply:
I'm having fun diving through the playground code base, figuring out how things are implemented. You seem to have a nice coherent mental model for the UI, so I'm happy to defer to you on how to expose these concerns.